components/form/overview.md

```diff
@@ -21,84 +21,9 @@ The <a href = "https://demos.telerik.com/blazor-ui/form/overview" target="_blank
     * Set the `EditContext` parameter to an [`EditContext` instance](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.forms.editcontext). The object instance from which the `EditContext` is created is important, especially when using [FormItem templates](slug:form-formitems-template) and [validation messages](slug:form-validation).
 1. (optional) To enable [form validation](slug:form-validation), add the `<FormValidation>` tag. Define a validator component inside, for example the [`DataAnnotationsValidator`](https://learn.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.components.forms.dataannotationsvalidator) that is part of .NET Core.
 
-<div class="skip-repl"></div>
-````RAZOR Model
-@* Provide a model to the Telerik Form *@
+>caption Basic Blazor Form
```

Comment:
Typically, captions throughout the Blazor docs use sentence-style casing. They also usually choose a phrase starting with a gerund (-ing form)

```suggestion
>caption Implementing a basic Blazor form
```